src/core/types.h

```diff
@@ -446,10 +446,12 @@ class Box {
 
     llvm::iterator_range<BoxIterator> pyElements();
 
+    size_t getHCAttrsOffset();
```

Comment:
oh shoot I forgot you had handled that by just aborting the rewrite when we run into those cases.  I'll take out that test.